spot_rl_experiments/spot_rl/utils/heuristic_nav.py

```diff
@@ -640,12 +641,19 @@ def scan_arm(
         # "ball"
         "caterpillar plush toy",
     ]
+    receptacle_name = str(receptacle_name.split(";")[-1:]).strip("[]'\"")
+    receptacle_name = receptacle_name.replace("_", " ")
+    print(receptacle_name)
+    height = calculate_height(receptacle_name)
     # Read gaze arm angles from config if None is passed
     if gaze_arm_angles is None:
         config = construct_config()
-        gaze_arm_angles = deepcopy(
-            config.GAZE_ARM_JOINT_ANGLES
-        )  # TODO: Toggle between short and tall receptacles
+
+        if height < config.HEIGHT_THRESHOLD:
```

Comment:
are we only checking for < height_threshold and default joint angles are good enough for taller receptacles?

Reply:
For taller receptacles, the object is seen in default arm angle configuration, whereas for shorter receptacles, the current angle doesn't see the object , causing the planner to get feedback as "no object found".